### ISSUE TITLE
packaging, installing and configuring setdefinitions

### ISSUE THREAD

**Issue by kosloot** [LanguageMachines/uctodata#5]
a new directory setdefinitions is added, and a generate-setdefinitions script too.

These are NOT packaged nor configured.
Is this intentional? So the tarball will NOT include these?

If not: configure.ac, Makefile.am and such have to be added/adapted
A dependency on Python3 is also new then.

**Comment by proycon** [comment]
Good point, but not too big of a problem yet: the fact that they are published here on github is good enough for now. FoLiA documents can just reference the setdefinitions here.

Regenerating them I can do manually for now (a dependency on python would be overkill anyhow). But perhaps we can automate things more eventually.
